src/components/LoginForm.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import logo from '../assets/logo.png';
+import { TextField, Button } from '@material-ui/core';
+import { makeStyles, createStyles } from '@material-ui/core/styles';
+
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    form: {
+      display: 'flex',
+      flexDirection: 'column',
+      width: '30%',
+      margin: 'auto',
+    },
+    img: {
+      width: '50%',
+    },
+    btn: {
+      backgroundColor: 'teal ',
+      color: 'white',
+      margin: '1.5em auto',
+      textTransform: 'capitalize',
+    },
+    input: {
+      marginTop: '2em',
+      paddingTop: '1em',
+    },
+  })
+);
+const LoginForm: React.FC = () => {
+  const classes = useStyles();
+  return (
+    <div>
+      <form noValidate autoComplete='off' className={classes.form}>
+        <img src={logo} alt='Pockets Logo' className={classes.img} />

```

Comment:
Could you adjust the logo position to be centered? Please refer to the screenshot in the issue.

![Screen Shot 2020-10-23 at 1 45 46 PM](https://user-images.githubusercontent.com/10347996/97036653-24844e80-1536-11eb-8e39-60d3f5aa57bb.png)